File: Lecture 3/StarterCode - Java/app/src/main/java/com/inovaeg/androidtrainingpharos2020/finaldemo/NetflixMovieOverviewActivity.java
package com.inovaeg.androidtrainingpharos2020.finaldemo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.inovaeg.androidtrainingpharos2020.R;

public class NetflixMovieOverviewActivity extends AppCompatActivity {

    private Boolean isAddedToMyList = false;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_netflix_movie_overview);

        //todo add click listener to change mylistbutton icon to checked when clicked and to add when clicked again
    }
}

File: Lecture 3/StarterCode - Kotlin/app/src/main/java/com/inovaeg/androidtrainingpharos2020/finaldemo/NetflixMovieOverviewActivity.kt
package com.inovaeg.androidtrainingpharos2020.finaldemo

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.inovaeg.androidtrainingpharos2020.R
import kotlinx.android.synthetic.main.activity_netflix_movie_overview.*

class NetflixMovieOverviewActivity : AppCompatActivity() {
    private val isAddedToMyList = false
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_netflix_movie_overview)

        info_btn.setOnClickListener {
            startActivity(Intent(this@NetflixMovieOverviewActivity, NetflixMoviePlayOrDownloadActivity::class.java))
        }

    }
}